### ISSUE TITLE
[a11y] /about/governance/policies/participation/ page accessibility issues

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15758]
# Axe-core® Accessibility Results for /en-US/about/governance/policies/participation/ (desktop) project

Page URL: [http://localhost:8000/en-US/about/governance/policies/participation/?automation=true](http://localhost:8000/en-US/about/governance/policies/participation/?automation=true)  

## axe-core found 4 violations

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | Heading levels should only increase by one | heading-order | Best practice | moderate | 2   |
| [2](#2) | Aside should not be contained in another landmark | landmark-complementary-is-top-level | Best practice | moderate | 1   |
| [3](#3) | Landmarks should have a unique role or role/label/title (i.e. accessible name) combination | landmark-unique | Best practice | moderate | 1   |

### Failed

1. Heading levels should only increase by one

[Learn more](https://dequeuniversity.com/rules/axe/4.10/heading-order?application=playwright)

heading-order

Best practice

Ensure the order of headings is semantically correct

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.side-reference:nth-child(2) > .side-reference-title<br>```<br><br>**Element source**<br><br>```<br><h4 class="side-reference-title">Our Products</h4><br>``` | Fix any of the following:<br><br>*   Heading order invalid |
| 2   | **Element location**<br><br>```<br>article > h4<br>```<br><br>**Element source**<br><br>```<br><h4>Version 3.1 – Updated January 16, 2020</h4><br>``` | Fix any of the following:<br><br>*   Heading order invalid |

2. Aside should not be contained in another landmark

[Learn more](https://dequeuniversity.com/rules/axe/4.10/landmark-complementary-is-top-level?application=playwright)

landmark-complementary-is-top-level

Best practice

Ensure the complementary landmark or aside is at top level

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.mzp-c-newsletter<br>```<br><br>**Element source**<br><br>```<br><aside class="mzp-c-newsletter t-love"><br>``` | Fix any of the following:<br><br>*   The null landmark is contained in another landmark. |

3. Landmarks should have a unique role or role/label/title (i.e. accessible name) combination

[Learn more](https://dequeuniversity.com/rules/axe/4.10/landmark-unique?application=playwright)

landmark-unique

Best practice

Ensure landmarks are unique

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.mzp-l-sidebar<br>```<br><br>**Element source**<br><br>```<br><aside class="mzp-l-sidebar"><br>``` | Fix any of the following:<br><br>*   The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable<br><br>**Related node:**<br><br>```<br>#newsletter-subscribe<br>``` |
